src/js/game/camera.js

```diff
@@ -809,6 +816,20 @@ export class Camera extends BasicSerializableObject {
 
         this.touchPostMoveVelocity = this.touchPostMoveVelocity.multiplyScalar(velocityFade);
 
+        // Check if the camera is being dragged but standing still: if not, zero out `touchPostMoveVelocity`.
+        if (this.currentlyMoving && this.desiredCenter === null) {
+            if (this.lastMovingPositionLastTick === this.lastMovingPosition) {
+                this.numTicksStandingStill += 1;
+            } else {
+                this.numTicksStandingStill = 0;
+            }
+            this.lastMovingPositionLastTick = this.lastMovingPosition;
```

Comment:
Could you use `.copy()` here? Otherwise you will share the same vector reference